src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/ImportServiceIntegrationTest.kt

```diff
@@ -48,11 +57,26 @@ internal class ImportServiceIntegrationTest(
   }
 
   @Test
-  internal fun `import should complete even when there are reports files with errors in them`() {
+  internal fun `given some report files contain errors the import should still complete successfully and not fail`() {
     service.importReportsOn(LocalDate.of(2020, 12, 1))
 
     verify(monitoringService).capture("moves: persisted 1 out of 2 for reporting feed date 2020-12-01.")
     verify(monitoringService).capture("people: persisted 3 out of 4 for reporting feed date 2020-12-01.")
     verifyNoMoreInteractions(monitoringService)
   }
+
+  @Test
+  internal fun `given a booked move which is subsequently cancelled in time, a journey for pricing should be created so the supplier can be paid`() {
```

Comment:
The data used for this was based on the move in question that was raised by the commercial team.  I have removed/changed any identifying data.